README.md

```diff
@@ -39,58 +49,57 @@ Load  `cfg80211` kernel module:
 sudo modprobe cfg80211
 ```
 
-Insert `vwifi` driver. In this case, vwifi will create two interfaces for us:
-```shell
-sudo insmod vwifi.ko ssid_list='[MyHomeWiFi][MyWifi_1][MyWifi_2]' interfaces=2
-```
-
-Check wiphy:
+Insert `vwifi` driver. In this case, vwifi will create three interfaces for us:
+(you can change the parameter "interfaces" on your favor)
 ```shell
-iw list
+sudo insmod vwifi.ko interfaces=3
```

Comment:
I dislike the parameter `interface` and the value `3`. Can you make it more meaningful?

Reply:
OK. I'll change `interfaces` to `station`. I think splitting interfaces to `station` and `ap` is a clearer way, but unfortunately, we cannot turn an interface to Host AP mode in initialization of `vwifi`, rather, it has to be call by hostapd. 

So, I think `station=3` would be a good way, because in initialization phase, we create three interfaces in station mode.

Reply:
Oh, You create the first 3 stations' virtual interface and force change one(`owl0`) to AP mode in the test script.

Reply:
Yes, it's not elegant, but it seems like the only way.